Angular/src/environments/environment.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
-    production: false,
-    googleAiKey: ""
-}
\ No newline at end of file
+  production: false,
```

Comment:
linting is applied everywhere. please be commit the changes only related to you exactly. i saw many places, spaces updates.